app/views/calculators/annuity/_calculator.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= render 'form', form: form %>
+<% if form.valid? %>
+  <%= render 'estimate',
+    estimate: form.estimate,
+    age: form.age.to_i,
+    pension_income: form.pension_income,
+    life_expectancy: form.life_expectancy.to_i,
+    valuation: form.estimate[:valuation]
```

Comment:
this is following the established pattern. I can see some benefit - not having to have the long accessor where it is used multiple times.

But agree it most likely isn't worth it in this case, but would suggest we revisit it when we make this production ready.